### ISSUE TITLE
Accessibility Images requiring Alt Text

### ISSUE THREAD

**Issue by ChalkandFeather** [hjtrhodes/bald-bible#66]
**Issue**
Images must have alternate text

**Steps to Reproduce:**
Navigate to homepage
Open console to view html and inspect images
run screen reader

**Expected Behaviour**
image to 

**Actual Behaviour**
Currently all feed images on the homepage missing <alt text>

<img width="1280" alt="Screenshot 2024-01-11 at 09 31 12" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/64e7d7f8-55e0-41c3-80e1-a521af98e468">

**Recommendations**
[https://dequeuniversity.com/rules/axe/4.8/image-alt?application=AxeChrome](url)

Accessibility priority: Critical
